exporter/azuredataexplorerexporter/metadata.yaml

```diff
@@ -6,4 +6,4 @@ status:
     beta: [traces, metrics, logs]
   distributions: [contrib]
   codeowners:
-    active: [asaharan, ag-ramachandran]
\ No newline at end of file
+    active: [asaharn, ag-ramachandran, tanmaya-panda1]
```

Comment:
Is @tanmaya-panda1 a member of OpenTelemetry?

Reply:
He is a team member. 
How can we check about his membership for OTel?

Reply:
Check here: https://github.com/orgs/open-telemetry/people